src/Symfony/Component/Form/Extension/Core/ChoiceList/ChoiceList.php

```diff
@@ -100,7 +108,7 @@ public function __construct($choices, array $labels, array $preferredChoices = a
      * @param array              $labels           The labels belonging to the choices.
      * @param array              $preferredChoices The choices to display with priority.
      */
-    protected function initialize($choices, array $labels, array $preferredChoices)
+    protected function initialize($choices, $labels, $preferredChoices)
```

Comment:
@stof is right. However, this class is not an API class, so technically this change is possible, as long as the change and the upgrade path is documented in the UPGRADE file (as per [our BC policy](http://symfony.com/doc/current/contributing/code/bc.html#changing-classes)). Can you add that information to the UPGRADE file please?